server/http.go

```diff
@@ -303,6 +303,10 @@ func (p *Plugin) postMeeting(creator *model.User, meetingID int, channelID strin
 		topic = defaultMeetingTopic
 	}
 
+	if !p.API.HasPermissionToChannel(creator.Id, channelID, model.PERMISSION_CREATE_POST) {
+		return errors.New("this channel is not accessible, you might not have permissions to write in this channel. Contact the administrator of this channel to find out if you have access permissions")
```

Comment:
@vicky-demansol We still need a unit test that covers the inside of this if statement

Reply:
I was add the unit test but the server response will return http code = 500, this response could be it returned as "forbidden" or "unauthorized"

Reply:
Hi @vicky-demansol, please make sure you mention the person you're replying to on GitHub issues/PRs, so we can stay up-to-date on discussions like this.

> I was add the unit test but the server response will return http code = 500, this response could be it returned as "forbidden" or "unauthorized"

Can we create a unit test just to cover that this line will run? If it take a lot of restructure to accomplish changing the response code, let's just make sure this error occurs in general.

Reply:
@micmister i was trying, if we want to get http code error need to use appError but i don't think this function can use in general, but this code on code coverage was covered